core/storage/user/gae_models_test.py

```diff
@@ -2218,3 +2218,98 @@ def test_create_raises_error_when_many_id_collisions_occur(self):
 
         with assert_raises_regexp_context_manager, get_by_id_swap:
             user_models.PseudonymizedUserModel.get_new_id('exploration')
+
+
+class UserAuthModelTests(test_utils.GenericTestBase):
+    """Tests for UserAuthModel."""
+
+    NONEXISTENT_USER_ID = 'id_x'
+    NONREGISTERED_GAE_ID = 'gae_id_x'
+    USER_1_ID = 'user_1_id'
+    USER_1_GAE_ID = ['gae_1_id']
+    USER_2_ID = 'user_2_id'
+    USER_2_GAE_IDs = ['gae_2_id_1', 'gae_2_id_2']
+
+    def setUp(self):
+        """Set up user models in datastore for use in testing."""
+        super(UserAuthModelTests, self).setUp()
+
+        user_models.UserAuthModel(
+            id=self.USER_1_ID,
+            gae_id=self.USER_1_GAE_ID,
+        ).put()
+        user_models.UserAuthModel(
+            id=self.USER_2_ID,
+            gae_id=self.USER_2_GAE_IDs,
+        ).put()
+
+    def test_get_export_policy(self):
```

Comment:
Test names should include both the conditions being tested & the expected outcome (see https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests).

In this case, consider: ``test_get_export_policy_policy_is_not_applicable``

Ditto for other tests below.

Reply:
Done now. Thanks for pointing to the resource.